### PR TITLE
mayhem integration

### PR DESCRIPTION
**Pull request by evanrichter**

### CODE REVIEW

File: crates/varcon-core/fuzz/fuzz_targets/cluster_parse.rs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#![no_main]
```
Comment (xansec) [inline crates/varcon-core/fuzz/fuzz_targets/cluster_parse.rs:1]: This seems to be part of varcon and not typos.

File: crates/typos/fuzz/fuzz_targets/tokenizer.rs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#![no_main]
```
Comment (xansec) [inline crates/typos/fuzz/fuzz_targets/tokenizer.rs:1]: This is fine.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: crates/varcon-core/fuzz/fuzz_targets/cluster_parse.rs (hunk 1 above)
Comment (evanrichter) [reply crates/varcon-core/fuzz/fuzz_targets/cluster_parse.rs:1]: while varcon is its own crate, the upstream is this repo: https://crates.io/crates/varcon-core

**Review by xansec — APPROVED** [review]
I'm fine with this, then. LGTM!

**Pull request merged**